contracts/seed/SeedFactory.sol

```diff
@@ -20,20 +20,16 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "./Seed.sol";
 import "../utils/CloneFactory.sol";
 
-
 /**
  * @title primeDAO Seed Factory
  * @dev   Enable primeDAO governance to create new Seed contracts.
  */
 contract SeedFactory is CloneFactory {
     using SafeMath for uint256;
 
-    Avatar    public avatar;
-    Seed      public parent;
-    bool      public initialized;
-
-    uint32  public constant PPM               = 1000000;   // parts per million
-    uint256 public constant PPM100            = 100000000; // ppm * 100
+    Avatar public avatar;
```

Comment:
Why is this called "avatar" (which implies a DAOstack DAO)?  Who is this really?  Just an "admin"?  I think we should change this.

Reply:
it is the DAO.

Reply:
We can keep `avatar`, as it is a more generic name not only `daostack` related, or we can rename it to `dao`

Reply:
it doesn't have to be a DAO either.  It can be any account.  Am I wrong?  Why assume it must be a DAO?  What is the role of this account?  Can we please document?

Reply:
Another option might be "owner" and make this contract ownable

Reply:
ok, agreed ownable will be done, in a separate PR